localstack/services/awslambda/lambda_api.py

```diff
@@ -217,7 +217,7 @@ def get_function_version(arn, version):
             'Version': version,
             'CodeSize': arn_to_lambda.get(arn).get_version(version).get('CodeSize'),
             'FunctionName': func_name,
-            'FunctionArn': arn,
+            'FunctionArn': arn + ":" + str(version),
```

Comment:
nit pick: maybe we can consistently use single quotes throughout this file.

Reply:
Oops missed one. Could this be linted?

Reply:
Good point, I'm adding a linter configuration now.